src/app/components/Messages/MessageGroup.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div>
+    <UserMessage
+      v-for="(message, index) in messages"
+      :message="message"
+      :key="message.message"
+      :first="index === 0"
```

Comment:
> I think we agreed that instead of the first message being handled differently, that the content that should only be displayed up front should be included in this component?

I'm not entirely sure what you mean - the first message is handled differently to display the username.

Reply:
Right - I'm saying that instead, the username should be part of this component, not the first message. I think we should be able to get away with putting the username and time on a separate line, as Slack and Discord do